bower.json

```diff
@@ -31,6 +31,6 @@
     "angular-scenario": ">=1.5.0"
   },
   "resolutions": {
-    "angular": "1.5.0"
+    "angular": ">=1.5.0"
   }
```

Comment:
Interesting. Should we do this to all our dependencies/packages or are there some that might cause issues if kept at the latest version?